1.1/learn/how-to-use-openapi-tools.md

```diff
@@ -27,33 +27,43 @@ Code generation from OpenAPI to Ballerina can produce `ballerina mock services`
 For build time client stub generation, annotation support is provided.
 
 ### Mock service from OpenAPI
-`ballerina openapi gen-service <moduleName>:<serivceName> 
+
+```ballerina
```

Comment:
This should be just ```. Adding `ballerina` tells to format it as Ballerina code

Reply:
Needs to be fixed in other, non-ballerina code places too

Reply:
Corrected.